replit.nix

```diff
@@ -1,5 +1,11 @@
 { pkgs }: {
+    env = {
+        OPENSSL_DIR = "${pkgs.openssl.dev}";
+        LIBCLANG_PATH = "${pkgs.libclang.lib}/lib";
+      };
   deps = [
+    pkgs.direnv
+    pkgs.just
     pkgs.gcc
     pkgs.mold
     pkgs.rustc
```

Comment:
This looks rather redundant, why can't we use our normal shell in replit?

Reply:
This lets you build it outside nix develop without throwing an error when the nix persistence flag is enabled. If you have nix persistence enabled it ignores the replit.nix and just uses the flake